inst/report/Report_KRI.Rmd

```diff
@@ -86,7 +86,7 @@ Report_OverviewText(
 for (i in seq_along(lCharts)) {
   MetricID <- names(lCharts)[[i]]
   if(!is.null(dfMetrics)){
-    MetricName <- dfMetrics$metric[dfMetrics$workflowid == MetricID]
+    MetricName <- dfMetrics$metric[dfMetrics$MetricID == MetricID]
```

Comment:
i'm getting an error: `Error in writeImpl(): ! Text to be written must be a length-one character vector`, so i think we might need to add a `%>% unique()` to the end of this to ensure MetricName is length one.

Reply:
just need to use this for the dfMetrics derivation now: 

```
dfMetrics <- wf_metrics %>% map_df(function(wf){
  wf$meta$vThreshold <- paste(wf$meta$vThreshold, collapse = ",")
  return(wf$meta)
})
```